### DISCUSSION TITLE
Merge doubletime & nightcore, halftime & daycore

### DISCUSSION THREAD

**Discussion by MitsukaiDokuro** [ppy/osu#13163] [Ideas]
Both doubletime and nightcore raise the tempo of a map. Please move their distinctive features "keep original pitch" and "added beats" to the correct place - to the mod options (like speed multiplier already is one). No more fighting which one is better. Same thing for halftime and daycore.

Maybe even merge all 4 to one mod, tempo change.
Maybe even merge that one with the beatmap stat override mod (hammer icon).

UPDATE: Please make  "added beats" mod option a volume slider instead, or add one on top

**Comment by mk56-spn** [comment]
bumping this because it seems to have gone under the radar and its a very good idea because there definetly does not need to be 4 mods for rate changing, 1 would be sufficient and moving it to the conversion section where it belongs with DA . dk how about i feel about merging it into DA but its plausible, 

as an addendum with the suggestions of changing sudden death to have acc / combo settings another parallel merger involving SD and PF would also greatly help clean up the mod section which right now imo feels unnecesarily bloated for the sake of legacy

**Reply by peppy** [reply]
It has not gone under the radar. This is a huge change and would require weeks of infrastructure work for backwards compatibility. Probably not going to happen from that angle, as good of an idea as it may be perceived as.

**Reply by mk56-spn** [reply]
thought that might be the case, hope it can happen someday maybe when the infrastrcuture is being reworked for new lazer mods ,although i understand its still added time

**Reply by peppy** [reply]
Probably not. Also reads bad to users (people know what DT is, renaming it to DA2xSPD or something silly is not good user experience).

**Reply by mk56-spn** [reply]
> Probably not. Also reads bad to users (people know what DT is, renaming it to DA2xSPD or something silly is not good user experience).

I forgot to answer this and they could be renamed to actually smart names I suppose.  Temp change? Ik you said it won't happen for other reasons but like that's what I'd name it in any case ( DT + NC + HT + DC). As for PF and SD "Pass condition" or something if the sort to allow it to be expanded into what many people wanted with like variable acc or combo conditions for passing

**Reply by frenzibyte** [reply]
Sudden Death and Perfect sound like they should be in a separate proposal discussion altogether.

**Reply by mk56-spn** [reply]
> Sudden Death and Perfect sound like they should be in a separate proposal discussion altogether.

Mmm. You are right.  But ppy seems incredibly adamant about the whole idea ( understandably so because of the infrastructure fiasco it'll be) so shall I make one or nah?

**Reply by frenzibyte** [reply]
I'm mentioning that in regards to allowing customisation on the "Sudden Death" mod. Not sure SD and PF would ever get merged.

**Reply by bdach** [reply]
existing related discussion https://github.com/ppy/osu/discussions/13120#discussioncomment-1452711

**Reply by mk56-spn** [reply]
Just read through this. Honestly this all needs to be broken down into separate issues adequately because rn it's a mess imo.

It mixes the categorization of sudden death options with the creation of a new mod section etc..but I definitely think it's important that this gets sorted before Lazer goes "online"

**Reply by bdach** [reply]
> before Lazer goes "online"

i have no idea what that is supposed to mean

**Reply by mk56-spn** [reply]
> > before Lazer goes "online"
> 
> i have no idea what that is supposed to mean

Leaderboard visible Score submission

**Reply by peppy** [reply]
@mk56-spn i beg taht you please post less on github. you're noise is counterproductive to our development efforts.

**Reply by mk56-spn** [reply]
Damn. Understood. Guess I'll shut up
